akka-docs/src/main/paradox/remoting.md

```diff
@@ -2,10 +2,13 @@
 
 For an introduction of remoting capabilities of Akka please see @ref:[Location Transparency](general/remoting.md).
 
+We recommend @ref:[Akka Cluster](cluster-usage.md) over using remoting directly. As remoting is the
+underlying module that allows for Cluster, it is still useful to understand details about it though.
```

Comment:
is this also included in remoting-artery docs?

Reply:
Ah, no, didn't think of that. Will add there as well.